cli/config/d2.config.app.js

```diff
@@ -1,6 +1,12 @@
 const config = {
     type: 'app',
 
+    // sets the `dir` HTML attribute for the app:
+    // options are 'ltr', 'rtl', and 'auto'. If set to 'auto', the direction
+    // will be inferred by the user's UI locale.
+    // The header bar direction will always be set by the locale.
+    direction: 'ltr',
```

Comment:
does this mean a direction has to be provided in the config, or it will fall back to ltr if not provided?

Reply:
These are the config defaults, so it will use 'ltr' if nothing is provided 👍

Reply:
(Removed to keep default env vars tidier)